### PR TITLE
Added support for _isOptional

### PR DESCRIPTION
**Pull request by brian-learningpool**

### CODE REVIEW

**Comment by taylortom** [comment]
+1

File: plugins/content/component/model.schema [hunk 1]
```diff
@@ -30,6 +30,15 @@
...
       "title": "Classes"
     },
+    "_isOptional": {
+      "type": "bool",
+      "default": "false",
+      "isSetting": true,
+      "inputType": {"type": "Boolean", "options": [true, false]},
+      "validators": [],
+      "title": "Is this optional?",
+      "help": "If set to 'true' this componenent does not have to be completed"
```
Comment (dennis-learningpool) [inline plugins/content/component/model.schema]: small spelling error componenent -> component

File: plugins/content/block/model.schema [hunk 2]
```diff
@@ -41,6 +41,15 @@
...
       "title": "Classes"
     },
+    "_isOptional": {
+      "type": "bool",
+      "default": "false",
+      "isSetting": true,
+      "inputType": {"type": "Boolean", "options": [true, false]},
+      "validators": [],
+      "title": "Is this optional?",
+      "help": "If set to 'true' this block, and components within it, does not have to be completed"
```
Comment (dennis-learningpool) [inline plugins/content/block/model.schema]: *does* not have to be completed -> *do* not have to be completed

**Comment by dennis-learningpool** [comment]
+1 - minor spelling corrections

File: plugins/content/block/model.schema (hunk 2 above)
Comment (brian-learningpool) [inline plugins/content/block/model.schema]: Thanks!

**Pull request merged**